File: python/worker.py
import asyncio
import logging
from datetime import timedelta

from workflow import *

from temporalio.client import Client
from temporalio.worker import Worker


async def main():
    logging.basicConfig(level=logging.INFO)

    # Start client
    client = await Client.connect("localhost:7233")

    worker = Worker(
        client,
        task_queue="bugfix-task-queue",
        workflows=[BugfixWorkflow],
        activities=[one, two, three],
        sticky_queue_schedule_to_start_timeout=timedelta(seconds=1),
    )
    logging.info("Starting worker.")
    await worker.run()


if __name__ == "__main__":
    asyncio.run(main())

File: typescript/src/worker.ts
import { NativeConnection, Worker } from '@temporalio/worker';
import * as activities from './activities';

async function run() {
  const connection = await NativeConnection.connect({
    address: 'localhost:7233',
  });

  const worker = await Worker.create({
    connection,
    namespace: 'default',
    taskQueue: 'bugfix-demo',
    workflowsPath: require.resolve('./workflows'),
    stickyQueueScheduleToStartTimeout: '1s',
    activities,
  });

  await worker.run();
}

run().catch((err) => {
  console.error(err);
  process.exit(1);
});
